app/posts/manage-teacher-training-applications/2020-05-18-access-control-history.md

```diff
@@ -0,0 +1,551 @@
+---
+title: Access control - feature design rationale
+description: Determining the best best model for managing organisational and user access controls and permissions
+date: 2020-05-18
+---
+
+{% from "figure/macro.njk" import appFigure with context %}
+
+Teacher training providers have to partner with ‘accredited bodies’ if they’re not an ‘accredited body’ themselves. Accredited bodies maintain the quality of teacher training providers.
+
+Different organisations need access to different information depending on what they’ve agreed with their partners.
+
+How can we go about achieving this in the Manage service?
+
+* [The initial problem statement](#the-initial-problem-statement)
```

Comment:
There’s a built in feature to generate tables of content automatically using `[[toc]]`, which means you only ever need to update headings in one place.

```suggestion
## Contents

[[toc]]
```

Reply:
Does TOC get me within page links and can I define the levels it displays down to?

Reply:
Currently it goes as deeps as level 3: https://github.com/DFE-Digital/bat-design-history/blob/fb463a3a0111f21f287b4fb7c51b07d63a991199/lib/libraries/markdown.js#L28

But we can change this if needed.